addOns/pscanrules/src/main/resources/org/zaproxy/zap/extension/pscanrules/resources/Messages.properties

```diff
@@ -282,35 +282,35 @@ pscanrules.serverheaderversioninfoleak.name = Server Leaks Version Information v
 
 pscanrules.stricttransportsecurity.compliance.malformed.content.desc = A HTTP Strict Transport Security (HSTS) header was found, but it contains some content that was not expected (perhaps curly quotes), the expectation is that the content be printable ASCII characters.
 pscanrules.stricttransportsecurity.compliance.malformed.content.name = Strict-Transport-Security Malformed Content (Non-compliant with Spec)
-pscanrules.stricttransportsecurity.compliance.malformed.content.refs = http://tools.ietf.org/html/rfc6797
+pscanrules.stricttransportsecurity.compliance.malformed.content.refs = https://tools.ietf.org/html/rfc6797
```

Comment:
This is a 301, better update to the final URL.

Reply:
> This is a 301, better update to the final URL.

done for all affected links

Reply:
This one was not changed yet https://github.com/zaproxy/zap-extensions/pull/5201/files#diff-5f76e38a9cd7b2a22adb1ee21b0d3c550e7251609be7879e71d209cebecaed86R313